db/migrate/20160115111829_chargeback_rate_detail_currency_not_nil.rb

```diff
@@ -0,0 +1,6 @@
+class ChargebackRateDetailCurrencyNotNil < ActiveRecord::Migration
+  # Migration in order to put a currency by default in rates that were seeded or added by a user before the addition of currencies
+  def up
+    ChargebackRateDetail.where(:chargeback_rate_detail_currency_id => nil).update_all(:chargeback_rate_detail_currency_id => ChargebackRateDetailCurrency.first.id)
```

Comment:
@tledesma I recreated one of the test errors on travis locally after running `git pull upstream master`:
`bundle exec rake test:migrations:setup test:migrations` from the base of the repo.

`ChargebackRateDetailCurrency.first` can be nil if we haven't seeded the currencies yet, which is what is happening on travis.  Seed will have to handle this outside of the migration.  

If we do a data migration, like this, we need to create a stub of the models involved. 
For example, ChargebackRateDetail and ChargebackRateDetailCurrency models can change from when you write this migration until someone runs it.  We need a snapshot of that model with only the parts you need for the migration.  In this case, just a tiny stub of the model.

See [here](https://github.com/ManageIQ/manageiq/blob/2401e1de4e63607c7bde8d32a6639c181e314c45/db/migrate/20140905020643_update_default_registration_channel_names.rb#L2-L4) for an example